webapp/src/domain/reports/TableReportConfigPage.tsx

```diff
@@ -298,6 +299,59 @@ export default function TableReportConfigPage() {
                                 />
                             </FormGroup>
                         </FormSection>
+
+                        <FormSection
+                            title={
+                                <>
+                                    Baseline
+                                    <Popover
+                                        headerContent="Baseline labels and function"
+                                        bodyContent={
+                                            <div>
```

Comment:
It would be useful to mention here that setting these is optional, and allows you to select another set of DataSets.